tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -3175,14 +3175,22 @@ voq/test_voq_counter.py::test_voq_queue_counter:
 voq/test_voq_fabric_capacity.py:
   skip:
     reason: "Skip test_voq_fabric_capacity on unsupported testbed."
+    conditions_logical_operator: "OR"
     conditions:
-      - "('t2' not in topo_name) or (asic_subtype not in ['broadcom-dnx']) or (asic_type in ['cisco-8000'])"
+      - "('t2' not in topo_name)"
+      - "(asic_subtype not in ['broadcom-dnx'])"
+      - "not any(i in platform for i in ['arista_7800', 'x86_64-nokia_ixr7250e'])"
+      - "(asic_type in ['cisco-8000'])"
 
 voq/test_voq_fabric_isolation.py:
   skip:
     reason: "Skip test_voq_fabric_isolation on unsupported testbed."
+    conditions_logical_operator: "OR"
     conditions:
-      - "('t2' not in topo_name) or (asic_subtype not in ['broadcom-dnx']) or (asic_type in ['cisco-8000'])"
+      - "('t2' not in topo_name)"
+      - "(asic_subtype not in ['broadcom-dnx'])"
+      - "not any(i in platform for i in ['arista_7800', 'x86_64-nokia_ixr7250e'])"
```

Comment:
@arista-nwolfe : can we add test gap to see if we have to reenable them again once we conclude if this is needed.

@tjchadaga / @vmittal-msft to conclude on that.

Reply:
As per discussion, we will keep this feature disabled on upper spine router as there are no fabric cards but keep working code available on the branch. So if requirement comes to enable them then it will be easy to do it. At that time, we can un-skip these tests.

Reply:
Updated to only skip on UpperSpineRouter